presto-spi/src/main/java/com/facebook/presto/spi/function/QualifiedFunctionName.java

```diff
@@ -33,11 +33,13 @@ private QualifiedFunctionName(CatalogSchemaName functionNamespace, String functi
         this.functionName = requireNonNull(functionName, "name is null").toLowerCase(ENGLISH);
     }
 
+    // Do not use this function, it is only used for test or json deserialization
```

Comment:
Maybe "This function should only be used for JSON deserialization. Do not use it directly."